changelog.d/20240518_021532_jacky.lam_helm_defaultstorage.md

```diff
@@ -0,0 +1,4 @@
+### Fixed
+
+- Helm chart: Add volumeMounts only if defaultStorage is enabled
```

Comment:
Could you please add a comment in the values file that in this case the user should add all the necessary mounts themselves?